prompting/agent.py

```diff
@@ -52,22 +52,24 @@ def __init__(
         system_template: str = None,
         persona: Persona = None,
         begin_conversation=True,
+        system_prompt: str | None = None,
     ):
-        if persona is None:
-            persona = create_persona()
-
         self.persona = persona
         self.task = task
         self.llm_pipeline = llm_pipeline
 
         if system_template is not None:
             self.system_prompt_template = system_template
 
-        self.system_prompt = self.system_prompt_template.format(
-            mood=self.persona.mood,
-            tone=self.persona.tone,
-            **self.task.__state_dict__(),  # Adds desc, subject, topic
-        )
+        self.system_prompt = system_prompt
```

Comment:
I don't quite understand the way this code was rearranged, it seems like the previous one looked cleaner

Reply:
This code always creates a system prompt with a specific mood, tone, or persona. For organics, we need a static system prompt, so to reuse the existing project code, I’ve added an additional check here